Renamed upgrade to update

nataliecarey opened the pull request:
Pre-Release available to test the upgrade flow.  I intentionally released one version before catering for the upgrade and one version after so the impact of the change could be tested.  

`13.10.2-rc.1` - the final version of this PR
`13.10.2-rc.0` - an earlier version of this PR which shows how the upgrade breaks (when upgrading to this version)

I've also added an environment variable to turn off checking the interim state check for install/upgrade because it fails every time on my machine.

nataliecarey commented on cypress/e2e/plugins/plugin-utils.js:
```diff
@@ -66,13 +68,15 @@ function performPluginAction (action, plugin, pluginName) {
-  cy.get(panelProcessingQuery)
-    .should('be.visible')
-    .contains(`${capitalize(action === 'upgrade' ? 'Upgrad' : action)}ing ...`)
+  if (Cypress.env('SKIP_PLUGIN_ACTION_INTERIM_STEP') !== 'true') {
```
Yeah, I've had problems for a while.  On my laptop and my home internet connection the install happens too quickly and it fails because it's in the complete state and it should be in the pending state.

nataliecarey commented on cypress/events/index.js:
```diff
@@ -61,6 +61,7 @@ module.exports = function setupNodeEvents (on, config) {
   config.env.password = process.env.PASSWORD
   config.env.projectFolder = path.resolve(process.env.KIT_TEST_DIR || process.cwd())
   config.env.tempFolder = path.join(__dirname, '..', 'temp')
+  config.env.SKIP_PLUGIN_ACTION_INTERIM_STEP = process.env.SKIP_PLUGIN_ACTION_INTERIM_STEP
```
Sure, I'll do that.

BenSurgisonGDS commented:
> Pre-Release available to test the upgrade flow. I intentionally released one version before catering for the upgrade and one version after so the impact of the change could be tested.
> 
> `13.10.2-rc.1` - the final version of this PR `13.10.2-rc.0` - an earlier version of this PR which shows how the upgrade breaks (when upgrading to this version)
> 
> I've also added an environment variable to turn off checking the interim state check for install/upgrade because it fails every time on my machine.

That's really odd.  I wonder if there is a timing issue or something.  Maybe it warrents more investigation in the future, but not in this PR.  I'll approve

nataliecarey commented:
> > Pre-Release available to test the upgrade flow. I intentionally released one version before catering for the upgrade and one version after so the impact of the change could be tested.
> > `13.10.2-rc.1` - the final version of this PR `13.10.2-rc.0` - an earlier version of this PR which shows how the upgrade breaks (when upgrading to this version)
> > I've also added an environment variable to turn off checking the interim state check for install/upgrade because it fails every time on my machine.
> 
> That's really odd. I wonder if there is a timing issue or something. Maybe it warrents more investigation in the future, but not in this PR. I'll approve

It's because I changed the URL, so it does the update on a URL including `/upgrade/` then after the update that URL doesn't exist anymore - now the `/upgrade/` and `/update/` behave the same.  That'll have to live for a long time so I've added a test for it.
